### ISSUE TITLE
The page menu keeps opening

### ISSUE THREAD

**Issue by SWatB-com** [fastaddons/ScrollAnywhere#52]
As you know, scrolling does not always work when pages are in the process of loading. This is normal. But, by releasing the right mouse button at such moments, the page menu constantly opens. Is it possible to do something so that in such cases the page menus do not open? For example: block the opening of the menu while the page is loading; or if there was an unsuccessful but still attempt to scroll the page. Or maybe tweak it somehow. Tired of the periodically pop-up menu of the page at the time of trying to scroll the page while it is loading.

In general, the program is very good. Thank you.

**Comment by fastaddons** [comment]
Interesting...
Currently the context menu is blocked only after mouse moved, at least twice, that is two mouse-move events were fired.
So if the page is crazy complex and CPU can't keep up, causing events not firing, that would cause the bug.

Another possibility is that settings didn't loaded in time and the script doesn't know you have right-click scrolling enabled so it won't block the context menu. But since settings has few KB or less, I would say it's unlikely.

**Can you send me some example page where I could reproduce this?**

Maybe I could block context menu if mouse-down and mouse-up events happened "too far" from each other.
But I'm not 100% sure. I wrote most of the code many years ago and I don't remember all details. And blocking context menu is pretty complex because I handle also some "special cases" when mouse is pressed inside "iframe" (nested page, for example some ad banner or comment section) and released outside it (and vice versa).

**Comment by SWatB-com** [comment]
This is true for any site when it is in a state of loading. Especially if there are a lot of images on it, because of which, it takes longer than usual to load. For example: https://ecco.ru/men/shoes/boots/?pr=1&c=cl

Taking this opportunity, I would like to say that your following programs are also very good: "Group Speed ​​Dial", and "Web Translate" – this one is even indispensable, at least I don't know a better program. Thank you.
